gnome/gstreamer1-gst-plugins-bad/Portfile

```diff
@@ -37,35 +37,18 @@ depends_build-append        port:pkgconfig \
 
 depends_lib                 port:gstreamer1-gst-plugins-base \
                             port:gobject-introspection \
-                            port:curl \
+                            port:faac \
```

Comment:
This might cause a licensing issue according to the variant it was behind, need to verify if that conflicts with LGPL2.0+